x-pack/plugins/security_solution/public/kubernetes/pages/translations.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const COLUMN_SESSION_START = i18n.translate(
+  'xpack.securitySolution.kubernetes.columnSessionStart',
+  {
+    defaultMessage: 'Date connected',
+  }
+);
+
+export const COLUMN_EXECUTABLE = i18n.translate(
+  'xpack.securitySolution.kubernetes.columnExecutable',
+  {
+    defaultMessage: 'Session Leader',
+  }
+);
+
+export const COLUMN_NODE = i18n.translate('xpack.securitySolution.kubernetes.columnNode', {
+  defaultMessage: 'Node',
+});
+
+export const COLUMN_ENTRY_USER = i18n.translate(
+  'xpack.securitySolution.kubernetes.columnEntryUser',
+  {
+    defaultMessage: 'Login User',
```

Comment:
Should probably label this Entry user. Also, you might want to confirm if these column names are sentence case, or each word is capitalized.

Reply:
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/16872649/174850869-3f1f0eba-1e11-46cc-9011-3e16848292c0.png">

I found that the alerts table column custom names capitalize each word

Reply:
Ok, yea. seems it's a bit insconsistent throughout kibana in general. I guess just stick with captilized words for now.

Reply:
Eui team suggested to use sentence case. will push a fix